docs/notebooks/util/plotting.py

```diff
@@ -133,6 +133,40 @@ def plot_function_2d(
     return fig, ax
 
 
+def plot_acq_function_2d(
+    acq_func,
```

Comment:
(I know we're not as consistent with typing in notebook code, but might as well)

```suggestion
    acq_func: AcquisitionFunction,
```